include/alpaka/core/ConcurrentExecPool.hpp

```diff
@@ -71,8 +71,7 @@ namespace alpaka::core
         };
 
         //! ITaskPkg.
-        // \NOTE: We can not use std::packaged_task as it forces the use of std::future
-        // but we additionally support boost::fibers::promise.
+        // \NOTE: We can not use std::packaged_task as it forces the use of std::future.
```

Comment:
This comment is no longer true, because without the fiber backend, we can actually use `std::packaged_task` and `std::future`. But no need to fix this now.